lib/screens/login/login_input_widget.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:lydiaryanfluttersurvey/resources/dimensions.dart';
+
+class LoginInputWidget extends StatelessWidget {
```

Comment:
This Input will be re-used on many screens: `Your contact information`, `Reset Password`..., so it's better to have a generic name. What do you think?
For example: `AppTextField`

| Your contact information  | Reset Password |
| ------------- | ------------- |
| ![CleanShot 2023-04-04 at 10 45 45](https://user-images.githubusercontent.com/17875522/229682147-59ccfb97-0404-42c2-9393-c8de10651cdd.png)  | ![CleanShot 2023-04-04 at 10 45 28](https://user-images.githubusercontent.com/17875522/229682159-32104304-6a67-4eb3-9745-5555767191e3.png)  |